packages/grid/_modules_/grid/locales/arSD.ts

```diff
@@ -0,0 +1,110 @@
+
+  // Columns selector toolbar button text
+  toolbarColumns: 'الأعمدة',
+  toolbarColumnsLabel: 'حدد الأعمدة',
+
+  // Filters toolbar button text
+  toolbarFilters: 'الفلاتر',
+  toolbarFiltersLabel: 'اظهر الفلاتر',
+  toolbarFiltersTooltipHide: 'إخفاء الفلاتر',
+  toolbarFiltersTooltipShow: 'اظهر الفلاتر',
+  toolbarFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} فلاتر نشطة` : `${count} فلاتر نشطة`,
+
+  // Export selector toolbar button text
+  toolbarExport: 'تصدير',
+  toolbarExportLabel: 'تصدير',
+  toolbarExportCSV: 'تنزيل كملف CSV',
+
+  // Columns panel text
+  columnsPanelTextFieldLabel: 'ايجاد عمود',
+  columnsPanelTextFieldPlaceholder: 'عنوان العمود',
+  columnsPanelDragIconLabel: 'إعادة ترتيب العمود',
+  columnsPanelShowAllButton: 'عرض الكل',
+  columnsPanelHideAllButton: 'اخفاء الكل',
+
+  // Filter panel text
+  filterPanelAddFilter: 'اضف فلتر',
+  filterPanelDeleteIconLabel: 'حذف',
+  filterPanelOperators: 'العاملين',
+  filterPanelOperatorAnd: 'و',
+  filterPanelOperatorOr: 'او',
+  filterPanelColumns: 'الأعمدة',
+  filterPanelInputLabel: 'قيمة',
+  filterPanelInputPlaceholder: 'قيمة التصفية',
+
+  // Filter operators text
+  filterOperatorContains: 'يحتوي على',
+  filterOperatorEquals: 'يساوي',
+  filterOperatorStartsWith: 'يبدأ ب',
+  filterOperatorEndsWith: 'ينتهي بـ',
+  filterOperatorIs: 'يكون',
+  filterOperatorNot: 'ليس',
+  filterOperatorAfter: 'يكون بعد',
+  filterOperatorOnOrAfter: 'في أو بعد',
+  filterOperatorBefore: 'يكون',
+  filterOperatorOnOrBefore: 'في أو قبل',
+  filterOperatorIsEmpty: 'فارغ',
+  filterOperatorIsNotEmpty: 'ليس فارغا',
+
+  // Filter values text
+  filterValueAny: 'أي',
+  filterValueTrue: 'صحيح',
+  filterValueFalse: 'خاطئة',
+
+  // Column menu text
+  columnMenuLabel: 'القائمة',
+  columnMenuShowColumns: 'إظهار الأعمدة',
+  columnMenuFilter: 'فلتر',
+  columnMenuHideColumn: 'إخفاء',
+  columnMenuUnsort: 'غير مرتب',
+  columnMenuSortAsc: 'الترتيب حسب ASC',
+  columnMenuSortDesc: 'الترتيب حسب DESC',
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} فلاتر نشطة` : `${count} فلاتر نشطة`,
+  columnHeaderFiltersLabel: 'اظهر الفلاتر',
+  columnHeaderSortIconLabel: 'ترتيب',
+
+  // Rows selected footer text
+  footerRowSelected: (count) =>
+    count !== 1
+      ? `${count.toLocaleString()} صفوف مختارة`
+      : `${count.toLocaleString()} صفوف مختارة`,
+
+  // Total rows footer text
+  footerTotalRows: 'إجمالي الصفوف:',
+
+  // Total visible rows footer text
+  footerTotalVisibleRows: (visibleCount, totalCount) =>
+    `${visibleCount.toLocaleString()} من ${totalCount.toLocaleString()}`,
+
+  // Checkbox selection text
+  checkboxSelectionHeaderName: 'تحديد',
+
+  // Boolean cell text
+  booleanCellTrueLabel: 'صحيح',
+  booleanCellFalseLabel: 'خاطئة',
+
+  // Used core components translation keys
+  MuiTablePagination: {},
+};
+
+export const arSD: Localization = getGridLocalization(arSDGrid, arSDCore);
```

Comment:
This locale doesn't exist in the [core](https://material-ui.com/guides/localization/#supported-locales) yet. Do you want to open another PR there to add the translations for the pagination or is it fine to use the English as default?

```suggestion
export const arSD: Localization = getGridLocalization(arSDGrid, enUSCore);
```

Reply:
updated